src/formulate.api/Submissions.cs

```diff
@@ -139,14 +139,25 @@ private static IValidationPersistence Validations
                 UmbracoContext = context.UmbracoContext,
                 UmbracoHelper = context.UmbracoHelper,
                 SubmissionId = Guid.NewGuid(),
-                ExtraContext = new Dictionary<string, object>()
+                ExtraContext = new Dictionary<string, object>(),
+                SubmissionCancelled = false
             };
 
 
             // Invoke submitting event (gives listeners a chance to change the submission).
             Submitting?.Invoke(submissionContext);
 
 
+			// Fail the form submission if SubmissionContext.SubmissionCancelled boolean is true
```

Comment:
@TFAstudio This is indented oddly.

Reply:
@TFAstudio FYI, I think the strange indent is because this line is mixing tabs and space. Should be spaces.